### PR TITLE
fix: redirectors and broken app links

### LINKED ISSUE #152: Too many broken links within sections in the documentation
Hello everybody!

Checking the documentation, I noticed that there are many broken directions (mainly due to the lack of the `.html` extension at the end of the url).

I can work on it and fix it!

### PR DESCRIPTION
**Pull request by lucalves**
Hi all!

In this PR:

- I've fixed some broken links;
- I updated some url redirectors;
- I fixed some indent in markdown files;

Closes #152.

### CODE REVIEW

**Comment by m1ga** [comment]
Thanks for the PR, can you please double check the anchor links. Some of them don't have `.html` and aren't working for me when the slash is removed.

**Comment by lucalves** [comment]
> Some of them don't have `.html` and aren't working for me when the slash is removed.

Could you send me the links? I tested it locally and had no problems.

I can help with that.

**Comment by m1ga** [comment]
e.g. this one:

https://github.com/tidev/titanium-docs/pull/153/files#diff-a8fdbf08bc735939cd6a10d9115e3ce84d70bec6794ec60e50a13bd4c7f29c54R390

**Comment by lucalves** [comment]
> e.g. this one:
> 
> https://github.com/tidev/titanium-docs/pull/153/files#diff-a8fdbf08bc735939cd6a10d9115e3ce84d70bec6794ec60e50a13bd4c7f29c54R390

Oh correct!

I'll check and upload a commit soon.

**Comment by lucalves** [comment]
@m1ga I've commited [#457c292](https://github.com/tidev/titanium-docs/pull/153/commits/457c2926eebf670062e09353af858868f0be1e8e), rechecking the links. Could you review?

**Comment by m1ga** [comment]
:+1: I'll check it but I'm not in the position to merge those. But since it looks valid I've already added the `hacktoberfest-accepted` tag so it counts for you.

In order to get it merged you'll have to sign the CLA: https://github.com/tidev/organization-docs/blob/main/AUTHORIZED_CONTRIBUTORS.md

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Creating_Your_First_Titanium_App.md [hunk 1]
```diff
@@ -420,7 +420,7 @@ In this case we’ll increase the font size used by each `<TableViewRow>` elemen
...
 
-To do this we'll take advantage of an Alloy feature called [platform-specific resources](/guide/Alloy_Framework/Alloy_Guide/Alloy_Concepts/#platform-specific-resources) that lets you define platform-specific view, controller, and style files. At build time, only those resources specific to the target platform are included. There are two parts to this:
+To do this we'll take advantage of an Alloy feature called [platform-specific resources](/guide/Alloy_Framework/Alloy_Guide/Alloy_Concepts#platform-specific-resources) that lets you define platform-specific view, controller, and style files. At build time, only those resources specific to the target platform are included. There are two parts to this:
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Creating_Your_First_Titanium_App.md]: html missing

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Android_SDK.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ Note that the typical filesystem location of this software can be found in the [
 
 ### Windows
 
-Note that the typical filesystem location of this software can be found in the [Windows Software Locations](/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Software_Locations_and_Environment_Variables/) section of these guides.
+Note that the typical filesystem location of this software can be found in the [Windows Software Locations](/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Software_Locations_and_Environment_Variables) section of these guides.
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Android_SDK.md]: .html missing

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Android_SDK.md [hunk 3]
```diff
@@ -143,7 +143,7 @@ Note that the typical filesystem location of this software can be found in the [
 
 11. Click the **Finish** button once Android Studio has finished downloading components.
 
-Refer to the [Windows Software Locations](/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Software_Locations_and_Environment_Variables/) section, and add the path of the **Android SDK Tools** directory to your system's `PATH`.
+Refer to the [Windows Software Locations](/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Software_Locations_and_Environment_Variables) section, and add the path of the **Android SDK Tools** directory to your system's `PATH`.
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Android_SDK.md]: .html missing

**Comment by lucalves** [comment]
> 👍 I'll check it but I'm not in the position to merge those. But since it looks valid I've already added the `hacktoberfest-accepted` tag so it counts for you.
> 
> In order to get it merged you'll have to sign the CLA: https://github.com/tidev/organization-docs/blob/main/AUTHORIZED_CONTRIBUTORS.md

I've sent a e-mail to him. 

Let's wait, thank you.

**Review by m1ga — COMMENTED** [review]
left some comments

**Review by m1ga — APPROVED** [review]
lgtm

**Comment by lucalves** [comment]
I still haven't received an email response about the CLA :(

@m1ga

**Comment by m1ga** [comment]
I'm sorry @lucalves  I've pinged him to have a look

**Comment by lucalves** [comment]
> I'm sorry @lucalves I've pinged him to have a look

Thank u so much

**Comment by joshualambert** [comment]
@lucalves Done. This is in your email now -- thanks so much for your contribution, friend! We're thrilled to have your support.

**Comment by m1ga** [comment]
Thank you @joshualambert ! All green now :+1:

**Comment by lucalves** [comment]
Can this be merged now?

**Comment by m1ga** [comment]
@lucalves there are other PRs waiting too. Once a Tidev person will look at it he can merge it. It can take a while. 

In case you do Hacktoberfest: it already counts since it has the label

**Comment by lucalves** [comment]
> @lucalves there are other PRs waiting too. Once a Tidev person will look at it he can merge it. It can take a while.
> 
> In case you do Hacktoberfest: it already counts since it has the label

Ok, thanks :)

**Review by ewanharris — APPROVED** [review]

**Comment by ewanharris** [comment]
Thanks @lucalves!

**Pull request merged**